modules/configuring-a-floating-action-button/proc-configuring-floating-action-button-as-a-dynamic-plugin.adoc

```diff
@@ -0,0 +1,158 @@
+:_mod-docs-content-type: PROCEDURE
+[id="proc-configuring-floating-action-button-as-a-dynamic-plugin_{context}"]
+= Configuring a floating action button as a dynamic plugin
+
+You can configure a dynamic plugin as a floating action button to perform actions or open an internal or external link.
+
+.Procedure
+
+. Add the `app-config-dynamic.yaml` content into `app.config-local.yaml`.
```

Comment:
Where this configuration will take place? Also, Do we have any sample or example content for users?